### PR TITLE
Load Image from Sass

### PR DESCRIPTION
**Pull request by jimyhdolores**
At the moment of using Sass to load an image in our project it gives us error, the responsibility I made in angular 8 and 9, here the discussion

https://github.com/single-spa/single-spa-angular/issues/202#issuecomment-643355876

### CODE REVIEW

File: website/versioned_docs/version-5.x/ecosystem-angular.md [hunk 1]
```diff
@@ -499,6 +499,37 @@ Then use it in your template:
 ```html
 <img [src]="'yoshi.png' | assetUrl" />
 ```
+##### Option 3
+
+upload via sass; This demonstration was done with the angular 8 and 9 versions, example:
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/ecosystem-angular.md:504]: What does "upload via sass" mean? This appears to be in error.

The first sentence of this option should describe the overall approach.

```suggestion
You can use Angular's [rebaseRootRelativeCssUrls](https://angular.io/cli/build) option to rewrite the URLs. This option has been deprecated in the latest versions of Angular.
```

File: website/versioned_docs/version-5.x/ecosystem-angular.md [hunk 2]
```diff
@@ -499,6 +499,37 @@ Then use it in your template:
 ```html
 <img [src]="'yoshi.png' | assetUrl" />
 ```
+##### Option 3
+
+upload via sass; This demonstration was done with the angular 8 and 9 versions, example: 
+
+```sass
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/ecosystem-angular.md:508]: Let's describe what this example is showing:

```suggestion
This option allows you to keep your css referencing asset urls, which will be rewritten to respect the webpack public path:

```sass
```

File: website/versioned_docs/version-5.x/ecosystem-angular.md [hunk 3]
```diff
@@ -499,6 +499,37 @@ Then use it in your template:
...
+.body-row {
+    background: url("/assets/images/person.jpg") no-repeat right bottom
+}
+```
+To be able to show your images using Sass URL we have to do some configuration in our angular.json file, should go to the attribute
+**architect > build > options**and add the attribute  **"rebaseRootRelativeCssUrls"** value "true".
+Now inside the assets attribute (which is an array) you should add a slash **"/"** to the value of the **"ouput"** attribute which outputs its external styles (this is optional); example
+
+ ```json
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/ecosystem-angular.md:517]: The `...` in the json makes the json invalid, which shows syntax highlighting errors in the markdown:

image

Please do the following:

1. Remove the `...`
2. Switch from tabs to spaces.

File: website/versioned_docs/version-5.x/ecosystem-angular.md [hunk 4]
```diff
@@ -499,6 +499,37 @@ Then use it in your template:
...
+			"options": {
+				...
+						"rebaseRootRelativeCssUrls": true,
+						"aot": true,
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/ecosystem-angular.md]: This example code should be specific to the `rebaseRootRelativeCssUrls` option. Is it required to do AOT compilation when using rebaseRootRelativeCssUrls? If not, let's remove this

```suggestion
```

File: website/versioned_docs/version-5.x/ecosystem-angular.md [hunk 5]
```diff
@@ -499,6 +499,37 @@ Then use it in your template:
...
+						"assets": [
```
Comment (internettrans) [inline website/versioned_docs/version-5.x/ecosystem-angular.md]: Remove this - it references Material UI which is not relevant for everyone.

**Review by internettrans — COMMENTED** [review]
Thanks for your work on this - a few comments to polish up the wording.

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 1 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:504]: Hello, well first of all, sorry, I am Peruvian and I still don't speak English very well; what i meant is "load an image with scss"

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 3 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:517]: You are right, I put those ellipsis hinting that there may be more attributes inside the json

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 5 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md]: Well in my case, I had to add a slash "/" in front of assets to be able to use the library, I suppose it is the same for other libraries; Maybe someone comes across this problem and can put it as a recommendation and carry out their tests

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 1 above)
Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:504]: No worries at all - I am glad you're contributing and I will help out with any minor grammatical errors. This approach is not specific to Sass, though, right? It also works with Less and CSS?

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 5 above)
Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angular.md]: Oh interesting, I didn't realize that. You're referring to the slash in `"output": "/assets"`? If that is needed, then yes let's keep this as an example.

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 1 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:504]: Test with sass and css and it works, I guess it applies for Less

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 5 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md]: Great thanks.

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 1 above)
Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:504]: 👍 let's not make the documentation refer specifically to Sass then.

**Comment by internettrans** [comment]
Another thing - Should we move this documentation from the [Assets](https://single-spa.js.org/docs/ecosystem-angular#angular-assets) section to the [Styles](https://single-spa.js.org/docs/ecosystem-angular#styles) section? I think many people will go there to learn how to handle the styles, as well.

**Comment by jimyhdolores** [comment]
I think you should go to the styles section, that's fine with me.

File: website/versioned_docs/version-5.x/ecosystem-angular.md (hunk 3 above)
Comment (jimyhdolores) [reply website/versioned_docs/version-5.x/ecosystem-angular.md:517]: I already removed the ellipsis

**Comment by internettrans** [comment]
Closing this in favor of #288

**Pull request closed without merging**